lib/identity_cache/query_api.rb

```diff
@@ -120,7 +120,14 @@ def check_association_scope(association_name)
         end
       end
 
-      def record_from_coder(coder) #:nodoc:
+      def record_from_coder(coder)
+        return unless coder
+        ActiveSupport::Notifications.instrument('identity_cache.hydration', class: coder[:class]) do
```

Comment:
having read the SO link now... I pretty much agree with that [comment though](http://stackoverflow.com/questions/6991135/what-does-it-mean-to-hydrate-an-object/20787106#comment31159656_6991192):
>Hydration refers to filling an existing object with data. It is not a synonym for serialization

Interesting discussions further down too. Also "ActiveRecord hydrating" doesn't seem to foreign to the "Rails community" either. In any case... what ever makes it the clearest to people consuming this... but again, this is not accounting for "serialization time before going over the wires" (which could be 0 with say an in-memory backed to IDC, yet (de)hydration would still be required). I hope I made it somewhat clearer what _this_ metric is about. Let me know what you guys think is the best naming though!